src/Tribe/Views/V2/Hooks.php

```diff
@@ -80,7 +81,7 @@ protected function add_actions() {
 		add_action( 'template_redirect', [ $this, 'action_initialize_legacy_views' ] );
 		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_customizer_in_block_editor' ] );
 		add_action( 'tec_events_views_v2_after_get_events', [ $this, 'action_set_title_events' ], 10, 2 );
-		add_action( 'init', [ $this, 'hide_event_end_time' ] );
+		add_action( 'tribe_views_v2_after_setup_loop', [ $this, 'hide_event_end_time' ], 2 );
```

Comment:
Since we're hooked to priority 2 here, Line 243 (uses the same hook) will fire, but this feels iffy.

Reply:
@Camwyn Good eye. 

Yes, the reason this is one of the only hooks that I could rely on to be the trigger to indicate we are rendering a view. This way we are not putting the init overhead on every request with other hooks, only runs when we are rendering a view.

I want the explicit separation too; one method to init, others for specific functions of the initialization.
```
// Does all the init / listeners.
$obj->hide_event_end_time();

// We need to register context.
$obj->set_context_for_views_v2_setup_loop( $view );
```